### PR TITLE
killDataSourceWhitelist in CoordinatorDynamicConfig accepts comma separated list of strings in addition to json array of strings so that coordinator console can do the updates correctly

### PR DESCRIPTION
**Pull request by drcrallen**
#3095

### CODE REVIEW

**Comment by gianm** [comment]
👍

**Comment by xvrl** [comment]
👍

File: server/src/main/java/io/druid/server/coordinator/CoordinatorDynamicConfig.java [hunk 1]
```diff
@@ -58,7 +65,21 @@ public CoordinatorDynamicConfig(
...
+      for (int i = 0; i < tmp.length; i++) {
+        String trimmed = tmp[i].trim();
+        if (!trimmed.isEmpty()) {
+          this.killDataSourceWhitelist.add(trimmed);
+        }
+      }
+    } else if (killDataSourceWhitelist instanceof Collection){
+      this.killDataSourceWhitelist = ImmutableSet.copyOf(((Collection) killDataSourceWhitelist));
+    } else {
+      this.killDataSourceWhitelist = ImmutableSet.of();
```
Comment (navis) [inline server/src/main/java/io/druid/server/coordinator/CoordinatorDynamicConfig.java]: nit. should we log something on this?

Comment (fjy) [inline server/src/main/java/io/druid/server/coordinator/CoordinatorDynamicConfig.java]: @navis this is a backport to 0.9.1 of something that was merged in master

Comment (navis) [inline server/src/main/java/io/druid/server/coordinator/CoordinatorDynamicConfig.java]: ah, ok.

**Pull request merged**